buku.py

```diff
@@ -2088,6 +2088,7 @@ def load_chrome_database(self, path, unique_tag, add_parent_folder_as_tag):
 
         roots = data['roots']
         for entry in roots:
+            # Needed to skip sync_transaction_version key from roots
```

Comment:
Please make it

    # Needed to skip 'sync_transaction_version' key from roots
Note the quotes around sync_transaction_version